Makefile

```diff
@@ -52,13 +54,13 @@ init:
 run:
 	mkdir -p logs
 	mkdir -p /tmp/cores/
-	$$(which openresty) -p $$PWD/
+	$(OR_EXEC) -p $$PWD/
```

Comment:
we need to specified fully path for config file, eg: `-c $$PWD/conf/nginx.conf`.

![image](https://user-images.githubusercontent.com/6814606/60780631-d9440200-a171-11e9-8a86-56575d5a777d.png)

Reply:
it may fail under Mac OSX.

Reply:
yes, `-c` is needed under Mac OSX. I have add a travis for it, and all check passed. Let's merge it alright?